README.md

```diff
@@ -1,11 +1,99 @@
 # Topiary
 
-![CI Status](https://github.com/tweag/topiary/actions/workflows/ci.yml/badge.svg)
+![CI Status][ci-badge]
 
 Topiary aims to be a uniform formatter for simple languages, as part of
-the [Tree-sitter](https://tree-sitter.github.io/tree-sitter/) ecosystem.
-It is named after the art of clipping or trimming trees into fantastic
-shapes.
+the [Tree-sitter] ecosystem. It is named after the art of clipping or
+trimming trees into fantastic shapes.
+
+Topiary is designed for formatter authors and formatter users. Authors
+can create a formatter for a language without having to write their own
+parser. Users benefit from uniform code style and, potentially, the
```

Comment:
Perhaps more importantly than the parser, is the whole engine as well.

I'm not sure the case for users is particularly good, but then again, we have little to say about that, I suppose. Maybe something along the line of “a single formatter for many languages with comparable style between languages”. After all, in our related work, we have a cornucopia of multi-language formatters. So it seems that's a value.

Reply:
Added in cd20b49a278a